src/main/thrift/octopus.thrift

```diff
@@ -0,0 +1,62 @@
+namespace scala com.gu.flexibleoctopus.model.thrift
+
+/** date times are reprsented as i64 - epoch millis */
+typedef i64 DateTime
+
+/** 
+* A single story bundle that maps directly to a piece of content in Composer 
+*/
+struct StoryBundle {
+
+    1: required i64 id;
+
+    2: required string composerId;
+
+    3: required string pubCode;
+
+    4: required string sectionCode;
+
+    5: required Article article;
+
+}
+
+/**
+* Represents a single article of a story bundle as generated by inCopy, 
+* such as the headline, standfirst, or body text
+*/
+struct Article {
+
+    // id for the article
+    1: required i64 id;
+
+    // id for the book section
+    2: required i64 bookSectionId;
```

Comment:
Looking at the headers spreadsheet, it's possible for the bundle to have a book section ID that is distinct from the articles. It seems like for that reason they need to be separate fields.

Reply:
There are many fields on the top level spreadsheet that are generated from the list of articles for convenience. This might be one. I have no idea what we would do in workflow or composer if these were different so question the value of having both when we should really just choose the right one. I also don't think we have any way of using the bookSectionId as we don't have reference mappings for the IDs, only the codes.